From 79f02a53316f90543d60269d7c06727c376f423b Mon Sep 17 00:00:00 2001
Patch-Source: https://github.com/dotnet/installer/issues/15040
From: Antoine Martin <dev@ayakael.net>
Date: Wed, 24 Aug 2022 18:41:08 +0000
Subject: [PATCH 1/1] disable generate composites

Introduction of https://github.com/dotnet/installer/pull/14549 has caused bug
when rebuilding aspnetcore using non-portable bits when generating composites.
This patch disables that function as it isn't necessary anyways. 

See https://github.com/dotnet/source-build/issues/3020

---
diff --git a/src/aspnetcore/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj b/src/aspnetcore/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj
index c8074844dc..50d9609f32 100644
--- a/src/aspnetcore/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj
+++ b/src/aspnetcore/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj
@@ -227,7 +227,6 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <CrossGenDependsOn Condition=" '$(CrossgenOutput)' == 'true' ">
       $(CrossGenDependsOn);
       _BatchCrossGenAssemblies;
-      _GenerateComposites;
     </CrossGenDependsOn>
     <!-- _GetBuildOutputFilesWithTfm executes in a separate context from Build, GenerateNuspec, and so on. -->
     <TargetsForTfmSpecificBuildOutput>
